quantum/keymap_extras/keymap_us_international_linux.h

```diff
@@ -0,0 +1,86 @@
+/* Copyright 2020
+ *
+ * This program is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License as published by
+ * the Free Software Foundation, either version 2 of the License, or
+ * (at your option) any later version.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with this program.  If not, see <http://www.gnu.org/licenses/>.
+ */
+
+/* 
+ * Linux and ChromeOS have a few differences vs. Windows in the
+ * AltGr map.
+ *
+ * See keymap_us_international.h for the core codes.
+ */
+
+#pragma once
+
+#include "keymap_us_international.h"
```

Comment:
In general I would prefer these headers to not depend on each other, for simplicity.

Reply:
Only reason I did it this way is because the Linux version is almost entirely an extension of the base version; seems odd to duplicate all of that code...

Reply:
I admit that at the moment the naming of these files makes it seem like `keymap_us_international.h` is the "base" set of defines, but for the most part they are simply whatever the Windows OSK told me. Eventually I would like to rename them all to be OS-specific.

Reply:
...but I just pushed the change that you request into the PR.